src/python/pants/jvm/resolve/coursier_fetch.py

```diff
@@ -138,8 +147,8 @@ def from_jvm_artifact_target(target: Target) -> Coordinate:
         version = target[JvmArtifactVersionField].value
         url = target[JvmArtifactUrlField].value
 
-        if url and url.startswith("file:/"):
-            raise CoursierError("Pants does not currently support `file:` URLS")
+        if url and url.startswith("file:"):
+            raise CoursierError("Pants does not support `file:` URLS. Use the `jar` field instead.")
```

Comment:
Maybe:  "The `url` field for the `jvm_artifact` target type does not support `file:` URLs. Instead, please use the `jar` field to specify the relative path to the local jar file within the project."